config/config.js

```diff
@@ -1,6 +1,6 @@
-window.API_HOST = 'http://localhost:4000'
+window.API_HOST = 'https://data.smartcolumbusos.com'
```

Comment:
This looks columbus specific. Is there a reason it's being changed from local host?

Reply:
Woops, I thought I pulled this request. Good call. 
I'll update but pulling anyway.